### PR TITLE
refactor(utxo-lib): move and rename addToPsbt, addChangeOutputToPsbt

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
See commits for details. Includes breaking changes.

Issue: BG-47824

### CODE REVIEW

**Review by davidkaplanbitgo — APPROVED** [review]
LGTM but @brandonblack should also take a look before merging

File: modules/utxo-lib/src/bitgo/wallet/WalletOutput.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+/**
+ * Add a verifiable wallet output to the PSBT. The output and all data
+ * needed to verify it from public keys only are added to the PSBT.
+ * Typically these are change outputs
+ *
+ * @param psbt the PSBT to add change output to
+ * @param rootWalletKeys keys that will be able to spend the output
+ * @param chain chain code to use for deriving scripts (and to determine script
+ *              type) chain is an API parameter in the BitGo API, and may be
+ *              any valid ChainCode
```
Comment (sangaman) [inline modules/utxo-lib/src/bitgo/wallet/WalletOutput.ts:17]: While we're moving code, we might as well get rid of this excess whitespace.

File: modules/utxo-lib/src/bitgo/wallet/Unspent.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
 import { Network } from '../..';
```
Comment (sangaman) [inline modules/utxo-lib/src/bitgo/wallet/Unspent.ts:1]: Should this file be `WalletUnspent.ts`? It seems like it'd go along with the naming convention for the other files within this folder, and would distinguish it from the `Unspent.ts` in the parent dir.

**Review by sangaman — APPROVED** [review]
LGTM, couple of minor suggestions.

File: modules/utxo-lib/src/bitgo/wallet/WalletOutput.ts (hunk 1 above)
Comment (sangaman) [reply modules/utxo-lib/src/bitgo/wallet/WalletOutput.ts:17]: Oh now with a slightly wider view I see that this whitespace is there to align the description of `chain` and not just some random whitespace. So we can leave as is.

File: modules/utxo-lib/src/bitgo/wallet/Unspent.ts (hunk 2 above)
Comment (OttoAllmendinger) [reply modules/utxo-lib/src/bitgo/wallet/Unspent.ts:1]: it probably should

**Pull request merged**